examples/timeline/other/onTimeout.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML>
+<!-- 
```

Comment:
I set `itemCount = 50` and I get a runtime exception `can not read property 'domProps' of null`, at line 735, [here](
https://github.com/yotamberk/vis/blob/4fe6d889218e24ce8e58d42d723cb5c6611b6f0e/lib/timeline/component/ItemSet.js#L733-L740). The demo just stops at that point.